store.go

```diff
@@ -26,12 +26,20 @@ var (
 	ErrTraceNotFound = errors.New("trace not found")
 )
 
+type TracesOpts struct {
+	Timespan Timespan
+}
+
+func DefaultTracesOpts() TracesOpts {
+	return TracesOpts{}
+}
```

Comment:
Good call!, I've added docstrings on https://github.com/sourcegraph/appdash/pull/127/files#diff-1ac94134b8964626aaac00790fa9b48fR29 for `TracesOpts` & `TracesOpts.Timespan`, and removed `DefaultTracesOpts(...)` in favor of using explicitly/readable `TracesOpts{}` as default options.